test_functions.c

```diff
@@ -0,0 +1,125 @@
+#include "ovector.h"
+#include "test_functions.h"
+#include <stdio.h>
+#include <assert.h>
+
+OVector vector;
+
+OVector vectorDeleted;
```

Comment:
don't create global objects - it's a bed idea and can shot in your leg some times